src/lib.rs

```diff
@@ -1303,7 +1307,62 @@ pub trait Drain {
         record: &Record,
         values: &OwnedKVList,
     ) -> result::Result<Self::Ok, Self::Err>;
-
+    /// Check if messages at the specified log level are enabled for this logger.
+    ///
+    /// Since disabled log levels are guaranteed to have their records ignored,
+    /// it's safe to skip logging messages at these level.
+    /// This can be used to avoid expensive computation of log message
+    /// arguments if the message would be ignored anyway.
+    ///
+    /// The logger is still free to ignore records even if the level is enabled,
+    /// an enabled level doesn't necessarily guarantee that the record will actually be logged.
+    /// ````
```

Comment:
Four ticks? I always use 3, is there any difference?

Reply:
I don't think there is but I'll change it for consistency ;)